Recipes/Recipes/All Recipes/Model/Category.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Category.swift
+//  Recipes
+//
+//  Created by Vermut xxx on 04.03.2024.
```

Comment:
```suggestion
//  Created by Vermut xxx on 04.03.2024.
```
шапку надо поправить, обычно форматер должен